vllm/attention/backends/flashinfer.py

```diff
@@ -60,19 +67,16 @@ class FlashInferMetadata(AttentionMetadata):
     # requests only.
     max_prefill_seq_len: int
 
-    use_cuda_graph: bool = False
+    use_cuda_graph: bool = True
```

Comment:
QQ: can we make it not kwarg?

Reply:
Here I'm following other backends such as https://github.com/vllm-project/vllm/blob/845a3f26f9706acafe8fa45ae452846d8cc3b97f/vllm/attention/backends/flash_attn.py#L120